.github/scripts/lib/cost-walker.ps1

```
@@ -200,55 +200,193 @@ function script:Add-CostWalkerAssistantEventAndSubagents {
-    if ($p -match '^([A-Za-z]):(.*)$') {
-        $driveLetter = $Matches[1].ToLowerInvariant()
-        $afterColon = $Matches[2].TrimStart('/')
-        $p = if ($afterColon) { "$driveLetter/$afterColon" } else { $driveLetter }
+    # Input guard (not part of the vendor rule): a path with no alphanumeric
+    # character at all derives an all-dash slug that can never name a real
+    # directory, and every downstream caller guards on emptiness rather than
+    # validity -- so '-' would pass those guards and start a walk on garbage
+    # where the pre-#908 implementation degraded to ''. No valid cwd reaches
+    # this branch; it exists so a malformed one fails the same way it used to.
+    if ($p -notmatch '[a-zA-Z0-9]') {
+        return ''
```

PSScriptAnalyzer (1.25.0)

[info] The cmdlet 'Get-CostTranscriptSlug' returns an object of type 'System.String' but this type is not declared in the OutputType attribute.

(PSUseOutputTypeCorrectly)